tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -231,21 +229,6 @@ var _ = Describe("VM backup and restore tests", Ordered, func() {
 			},
 		}, nil),
 
-		Entry("no-application CSI+datamover backup and restore, powered-off CirrOS VM", Label("virt"), VmBackupRestoreCase{
```

Comment:
just to make sure (PR description got me confused), as discussed in Scrum, this is a case that currently does not work and need Velero update to work in the future?

Reply:
Yes, this is a case that does not work right now. I added this test in the first place because I thought Velero had a fix, but it does not. I think I heard there is a Jira ticket or some other plan to fix it in the future though.